### PR TITLE
add tools to interactively modif a simple mesh

### PR DESCRIPTION
**Pull request by kerautret**

### CODE REVIEW

File: visualisation/compClass/ViewerMesh.cpp [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+  text += "Use the mouse to move the camera around the object. ";
+  text += "You can respectively revolve around, zoom and translate with the three mouse buttons. ";
+  text += "Left and middle buttons pressed together rotate around the camera view direction axis<br><br>";
+  text += "Pressing <b>Alt</b> and one of the function keys (<b>F1</b>..<b>F12</b>) defines a camera keyFrame. ";
+  text += "Simply press the function key again to restore it. Several keyFrames define a ";
+  text += "camera path. Paths are saved when you quit the application and restored at next start.<br><br>";
+  text += "Press <b>F</b> to display the frame rate, <b>A</b> for the world axis, ";
+  text += "<b>Alt+Return</b> for full screen mode and <b>Control+S</b> to save a snapshot. ";
+  text += "See the <b>Keyboard</b> tab in this window for a complete shortcut list.<br><br>";
+  text += "Double clicks automates single click actions: A left button double click aligns the closer axis with the camera (if close enough). ";
```
Comment (dcoeurjo) [inline visualisation/compClass/ViewerMesh.cpp]: automate

File: visualisation/compClass/ViewerMesh.cpp [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+
+
+template < typename Space, typename KSpace>
+void
+ViewerMesh< Space, KSpace>::keyPressEvent ( QKeyEvent *e )
+{
+  
+  bool handled = false;
+  if( e->key() == Qt::Key_Plus){
```
Comment (dcoeurjo) [inline visualisation/compClass/ViewerMesh.cpp]: coding style;) ?

**Comment by kerautret** [comment]
@dcoeurjo thanks for the remarks, done

**Comment by kerautret** [comment]
thanks @dcoeurjo for the remarks, merging

**Pull request merged**